webpack/webpack-plugin-utilities/package.json

```diff
@@ -34,6 +34,6 @@
     "@rushstack/heft": "workspace:*",
     "local-node-rig": "workspace:*",
     "@types/tapable": "1.0.6",
-    "webpack": "~5.82.1"
+    "webpack": "~5.95.0"
```

Comment:
Update to `^5.95.0` and call this out in the changelog? Should be a minor bump too.